.gitignore

```diff
@@ -149,3 +149,6 @@ GSYMS
 /tools/cub-1.8.0/
 /tools/cub
 /tools/python/
+
+/src/**/CMakeLists.txt
```

Comment:
I'm afraid filtering this out is not a good idea. It will confused people.

Reply:
Apparently, you are confused. `/src/**/CMakeLists.txt` are all generated on the fly.